### PR TITLE
Micke learns about Rails #1

### PR DESCRIPTION
**Pull request by pIkEL618**
Separate jsonbuilder stuff to using Partials.
Fix show endpoint in picture_parts controller.

parts gif

### CODE REVIEW

File: app/controllers/picture_parts_controller.rb [hunk 1]
```diff
@@ -4,13 +4,11 @@ class PicturePartsController < ApplicationController
...
   def index
     @picture_parts = PicturePart.all
-
-    render json: @picture_parts
   end
 
   # GET /picture_parts/1
   def show
-    @picture_part
+    @picture_part
```
Comment (Duwo) [inline app/controllers/picture_parts_controller.rb]: Onödigt space

File: app/controllers/picture_parts_controller.rb [hunk 2]
```diff
@@ -20,6 +18,7 @@ def random
 
     @picture_part = PicturePart.where(image_part: nil).first
   end
+
```
Comment (Duwo) [inline app/controllers/picture_parts_controller.rb]: Extra rad råkade komma med

File: app/views/picture_parts/show.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1,2 @@
+json.partial! "picture_parts/picture_part", part: @picture_part
+json.image_part @picture_part.image_part
```
Comment (Duwo) [inline app/views/picture_parts/show.json.jbuilder]: Ska denna vara med?

Comment (pIkEL618) [reply app/views/picture_parts/show.json.jbuilder]: yes gif

**Pull request merged**